dependencies.yaml

```diff
@@ -370,6 +377,21 @@ dependencies:
           # Align nvcomp version with rapids-cmake
           - nvcomp==4.0.1
```

Comment:
Remove in favor of `depends_on_nvcomp`. Same goes throughout this file -- some reworking of the file lists may be needed, too.
```suggestion
```